pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-actions-and-events.md

```diff
@@ -118,6 +118,10 @@ For example, the following is possible in AMP:
 
 [/tip]
 
+## Event/Action trust compatibility
+
+Similar to native web events and actions, AMP follows a trust model that insures that critical actions are only triggered by highly trusted events, (usually meaning that the event resulted directly from a user interaction such as a tap or click). Currently AMP supports two levels of trust as defined by `ActionTrust` which are `ActionTrust.HIGH` and `ActionTrust.LOW`. An action can only be triggered by an event of equal or higher trust level than required by the action. Refer to the tables below to compare event and action trust levels.
```

Comment:
What do you mean by "native web events and actions"?

Reply:
suggestion: replace `, (usually meaning that the event resulted directly from a user interaction such as a tap or click)` with `. High trust events are usually triggered by a user interaction such as a tap or click.`